demos/monge_ampere_3d.py

```diff
@@ -0,0 +1,94 @@
+# Monge-Ampère mesh movement in three dimensions
+# ====================================================================
```

Comment:
I was under the impression the number of `=` signs needed to match the title to avoid sphinx warnings.